harvdev_utils/examine_archives/search_archives.py

```diff
@@ -101,10 +110,10 @@ def search_proforma(grep_dir_string, field=None, value=None):
         if 'Is a directory' in line:
             continue
         if field:  # make sure it has a value
-            parts = line.strip().split(':')
-            if parts[2]:
+            (filename, prof_desc, prof_value) = line.strip().split(':')
```

Comment:
nice, but if there are ever more than three parts (e.g., gene symbol has `:` char, which happens), I think you'll get a `ValueError: too many values to unpack (expected 3)` error.